src/serlio/serlioPlugin.h

```diff
@@ -19,12 +19,23 @@
 
 #pragma once
 
+#include <cstdint>
+
 #ifdef _WIN32
-#	ifdef SRL_TEST_EXPORTS
-#		define SRL_TEST_EXPORTS_API __declspec(dllexport)
-#	else
-#		define SRL_TEST_EXPORTS_API
-#	endif
+#   ifdef SRL_TEST_EXPORTS
+#      define SRL_TEST_EXPORTS_API __declspec(dllexport)
+#   else
+#      define SRL_TEST_EXPORTS_API
+#   endif
 #else
-#	define SRL_TEST_EXPORTS_API __attribute__ ((visibility ("default")))
+#   define SRL_TEST_EXPORTS_API __attribute__ ((visibility ("default")))
 #endif
+
+namespace SerlioNodeIDs {
+
+// our registered node id range is: 0x00132980 - 0x001329bf
+constexpr std::uint32_t ID_RANGE_START = 0x00132980;
+constexpr std::uint32_t MODIFIER_NODE_SUB_ID = 10;
+constexpr std::uint32_t MATERIAL_NODE_SUB_ID = 20;
```

Comment:
are these two sub IDs on purpose in decimal form?

Reply:
doesn't really matter, but probably would be more consistent, i'll fix it

Reply:
done